cmd/config/olm/olm.yml

```diff
@@ -0,0 +1,87 @@
+---
+global:
+  gc: {{.GC}}
+  gcMetrics: {{.GC_METRICS}}
+  measurements:
+    - name: jobSummary
```

Comment:
Remove this line, as `jobSummary` is not a measurement but a document/metric that is automatically generated.